### DISCUSSION TITLE
Third heating circuit not listed

### DISCUSSION THREAD

**Discussion by loicguiraud** [dominikamann/oekofen-pellematic-compact#8] [Ideas]
I only get 2 of my 3 heating circuit.
Is it possible to add a the third one ?
It's listed in the json file

**Comment by dominikamann** [comment]
Hi @loicguiraud sure. Can you please provide me your JSON-File. Then I'm able to test it. Thanks.

**Comment by loicguiraud** [comment]
Here it is, it's the HK3 which is missing : 
{
 "system":{
  "L_ambient":45, 
  "L_errors":0, 
  "L_usb_stick":0, 
  "L_existing_boiler":0
 },
 "weather":{
  "L_temp":60, 
  "L_clouds":6, 
  "L_forecast_temp":70, 
  "L_forecast_clouds":71, 
  "L_forecast_today":0, 
  "L_starttime":940, 
  "L_endtime":1610, 
  "L_source":"http://www.openweathermap.org", 
  "L_location":"Lozanne|FR|2997295", 
  "cloud_limit":55, 
  "hysteresys":-40, 
  "offtemp":-100, 
  "lead":120, 
  "refresh":0, 
  "oekomode":0
 },
 "forecast":{
  "L_w_0":"mer., 8 févr. 17:58|6|6|2 km/h|01n|800|C|07:54|17:56", 
  "L_w_1":"mer., 8 févr. 19:00|5|9|3 km/h|01n|800|C", 
  "L_w_2":"mer., 8 févr. 22:00|4|12|1 km/h|02n|801|C", 
  "L_w_3":"jeu., 9 févr. 01:00|2|10|4 km/h|01n|800|C", 
  "L_w_4":"jeu., 9 févr. 04:00|0|7|3 km/h|01n|800|C", 
  "L_w_5":"jeu., 9 févr. 07:00|0|12|2 km/h|02n|801|C", 
  "L_w_6":"jeu., 9 févr. 10:00|3|76|0 km/h|04d|803|C", 
  "L_w_7":"jeu., 9 févr. 13:00|7|57|3 km/h|04d|803|C", 
  "L_w_8":"jeu., 9 févr. 16:00|7|86|4 km/h|04d|804|C", 
  "L_w_9":"jeu., 9 févr. 19:00|3|76|1 km/h|04n|803|C", 
  "L_w_10":"jeu., 9 févr. 22:00|1|12|5 km/h|02n|801|C", 
  "L_w_11":"ven., 10 févr. 01:00|0|12|5 km/h|02n|801|C", 
  "L_w_12":"ven., 10 févr. 04:00|-1|9|5 km/h|01n|800|C", 
  "L_w_13":"ven., 10 févr. 07:00|-1|6|4 km/h|01n|800|C", 
  "L_w_14":"ven., 10 févr. 10:00|2|4|4 km/h|01d|800|C", 
  "L_w_15":"ven., 10 févr. 13:00|6|7|6 km/h|01d|800|C", 
  "L_w_16":"ven., 10 févr. 16:00|7|45|7 km/h|03d|802|C", 
  "L_w_17":"ven., 10 févr. 19:00|2|27|4 km/h|03n|802|C", 
  "L_w_18":"ven., 10 févr. 22:00|1|5|4 km/h|01n|800|C", 
  "L_w_19":"sam., 11 févr. 01:00|0|14|3 km/h|02n|801|C", 
  "L_w_20":"sam., 11 févr. 04:00|0|25|2 km/h|03n|802|C", 
  "L_w_21":"sam., 11 févr. 07:00|-1|34|2 km/h|03n|802|C", 
  "L_w_22":"sam., 11 févr. 10:00|3|17|5 km/h|02d|801|C", 
  "L_w_23":"sam., 11 févr. 13:00|8|16|7 km/h|02d|801|C", 
  "L_w_24":"sam., 11 févr. 16:00|8|14|8 km/h|02d|801|C"
 },
 "hk1":{
  "L_roomtemp_act":0, 
  "L_roomtemp_set":80, 
  "L_flowtemp_act":347, 
  "L_flowtemp_set":80, 
  "L_comfort":0, 
  "L_state":2, 
  "L_statetext":"Priorité ECS active",
  "L_pump":0, 
  "remote_override":0, 
  "mode_auto":1, 
  "time_prg":0, 
  "temp_setback":180, 
  "temp_heat":220, 
  "temp_vacation":150, 
  "name":"Radiateurs 1", 
  "oekomode":0
 },
 "hk2":{
  "L_roomtemp_act":0, 
  "L_roomtemp_set":80, 
  "L_flowtemp_act":381, 
  "L_flowtemp_set":80, 
  "L_comfort":0, 
  "L_state":2, 
  "L_statetext":"Priorité ECS active",
  "L_pump":0, 
  "remote_override":0, 
  "mode_auto":1, 
  "time_prg":0, 
  "temp_setback":210, 
  "temp_heat":220, 
  "temp_vacation":150, 
  "name":"Plancher 1", 
  "oekomode":0
 },
 "hk3":{
  "L_roomtemp_act":0, 
  "L_roomtemp_set":80, 
  "L_flowtemp_act":344, 
  "L_flowtemp_set":80, 
  "L_comfort":0, 
  "L_state":2, 
  "L_statetext":"Priorité ECS active",
  "L_pump":0, 
  "remote_override":0, 
  "mode_auto":1, 
  "time_prg":0, 
  "temp_setback":180, 
  "temp_heat":200, 
  "temp_vacation":150, 
  "name":"Radiateurs 2", 
  "oekomode":0
 },
 "ww1":{
  "L_temp_set":600, 
  "L_ontemp_act":586, 
  "L_offtemp_act":586, 
  "L_pump":1, 
  "L_state":16400, 
  "L_statetext":"t dans prog horaire|Demande marche on",
  "time_prg":0, 
  "sensor_on":0, 
  "sensor_off":0, 
  "mode_auto":1, 
  "mode_dhw":1, 
  "heat_once":0, 
  "temp_min_set":300.0, 
  "temp_max_set":600, 
  "name":"Eau chaude", 
  "smartstart":0, 
  "use_boiler_heat":0, 
  "oekomode":0
 },
 "pe1":{
  "L_temp_act":652, 
  "L_temp_set":700, 
  "L_ext_temp":-32768, 
  "L_frt_temp_act":4049, 
  "L_frt_temp_set":5065, 
  "L_frt_temp_end":6500, 
  "L_br":0, 
  "L_ak":0, 
  "L_not":1, 
  "L_stb":1, 
  "L_modulation":66, 
  "L_runtimeburner":1000, 
  "L_resttimeburner":8000, 
  "L_currentairflow":45, 
  "L_lowpressure":1164, 
  "L_lowpressure_set":1162, 
  "L_fluegas":42, 
  "L_uw_speed":56, 
  "L_state":4, 
  "L_statetext":"Combustion",
  "L_type":9, 
  "L_starts":143, 
  "L_runtime":300, 
  "L_avg_runtime":125, 
  "L_uw_release":600, 
  "L_uw":56, 
  "L_storage_fill":5898, 
  "L_storage_min":400, 
  "L_storage_max":6000, 
  "L_storage_popper":0, 
  "storage_fill_today":0, 
  "storage_fill_yesterday":0, 
  "mode":1
 },
   "error":{
   }
}

**Comment by dominikamann** [comment]
@loicguiraud i implemented it. Details -> https://github.com/dominikamann/oekofen-pellematic-compact/discussions/9
